src/providers/supabase/authProvider.ts

```diff
@@ -55,6 +55,28 @@ export namespace getIsInitialized {
 export const authProvider: AuthProvider = {
     ...baseAuthProvider,
     checkAuth: async params => {
+        // Users are on the set-password page, nothing to do
```

Comment:
Why is checkAuth called on these pages?

Reply:
TBH I don't know, but I figured applying the same implementation as what ra-supabase does couldn't be a bad thing.